test/component/cypress/specs/ExportWins/CustomerFeedback.cy.jsx

```diff
@@ -41,26 +41,42 @@ const dummyExportWin = (prefix, booleans) => ({
   },
 })
 
+const dummyCompany = () => ({
+  id: '123',
+})
+
 describe('ExportWins/CustomerFeedback', () => {
   ;[
-    {testTitle: 'All true', win: dummyExportWin('all-true', true)},
-    {testTitle: 'All false', win: dummyExportWin('all-false', false)},
-  ].forEach(({testTitle, win}) => {
+    {
+      testTitle: 'All true',
+      win: dummyExportWin('all-true', true),
+      company: dummyCompany(),
+    },
+    {
+      testTitle: 'All false',
+      win: dummyExportWin('all-false', false),
+      company: dummyCompany(),
+    },
```

Comment:
I would just hardcode `{id: 123}` here as the id can be anything and the `dummyCompany` function makes one thing that there's hole lot more to it than there really is.

The `dummyExportWin` is a function because it's parametrized.

```suggestion
    {
      testTitle: 'All true',
      win: dummyExportWin('all-true', true),
      company: {id: 123},
    },
    {
      testTitle: 'All false',
      win: dummyExportWin('all-false', false),
      company: {id: 123},
    },
```